### ISSUE TITLE
Standaardlijsten voor nieuwe gebruikers aanmaken

### ISSUE THREAD

**Issue by ocrasborn** [signfuse/corpusNGT#59]
Ik merk bij mezelf dat het werken met de lijsten nog niet helemaal intuïtief gaat: waar voeg je items aan een lijst toe? Het kan niet bij de lijsten zelf, die geven alleen weer wat er in zit.
Ik denk dat het zou helpen, en wat frustratie bij de gebruiker zou schelen, als we een paar ijsten van Video's, Gebaarders, en Zoekfuncties zouden aanleveren aan elke nieuwe gebruiker. Bv.:
- Gebaarders: "Alle mannen"
- Gebaarders: "Alle vrouwen"
- Video's: "Duidelijke meningen"
- Zoekfuncties: "Alle Groningse 50+'ers"
Ik kan die zelf vullen met items, maar @signfuse , jij zou iets moeten maken waardoor deze vier lijsten voor alle gebruikers standaard aangeleverd worden.

**Comment by signfuse** [comment]
Ja, ga ik doen. Kan je met één van je testprofielen een template maken en dit profiel dan aan mij doorgeven?
